userspace/libscap/scap_procs.c

```diff
@@ -558,6 +558,21 @@ static int32_t scap_proc_add_from_proc(scap_t* handle, uint32_t tid, int parentt
 	}
 
 	//
+	// Gathers the exepath
+	//
+	snprintf(tinfo->exepath, sizeof(tinfo->exepath), "%s", target_name);
```

Comment:
Thanks for the review, I will fix that by null-terminate `target_name` before using it.